client/Caddyfile

```diff
@@ -0,0 +1,31 @@
+{$CADDY_HOST} { # Set host with environment variable
+    root /www/client
+
+    tls {$CADDY_TLS} # Set TLS settings, either "off" or an email address
+
+    proxy /api server:4567 # Proxy requests to /api to the server container
+
+    # Rewrite paths that don't start with /api to / for the Angular app
+    rewrite {
+      if {path} not_match ^/api
+      to {path} {path}/ /
+    }
+
+    expires {
+        match .js$ 1y
+        match .css$ 1y
```

Comment:
Does this mean that JS and CSS will be cached for a _year_? I can see how that might make sense for a more stable product, but that doesn't seem good for us. Or am I just misunderstanding something?

Reply:
When the client is built by webpack the js and css files have a hash of their contents added to their filenames so they are practically immutable. This allows us to cache them aggressively because new versions will be different filenames.

Reply:
I didn't know about the renaming trick. I guess with that it doesn't make much difference. It might be good to have a comment in `Caddyfile` that says something to that effect (maybe with a link to some documentation on that) so people aren't confused by that in the future.